File: t-tasks/src/main/java/com/teo/ttasks/api/PeopleApi.java
package com.teo.ttasks.api;

import com.teo.ttasks.api.entities.PersonResponse;

import io.reactivex.Flowable;
import retrofit2.http.GET;

public interface PeopleApi {

    @GET("people/me")
    Flowable<PersonResponse> getCurrentUserProfile();
}

File: t-tasks/src/main/java/com/teo/ttasks/api/PeopleApi.kt
package com.teo.ttasks.api

import com.teo.ttasks.api.entities.CoverPhotosResponse
import io.reactivex.Single
import retrofit2.http.GET

interface PeopleApi {

    @GET("people/me?personFields=coverPhotos")
    fun getCurrentPersonCoverPhotos(): Single<CoverPhotosResponse>
}
